### ISSUE TITLE
Sync issue after winning chia block in pool?[Bug] 

### ISSUE THREAD

**Issue by deanh2k** [Chia-Network/chia-blockchain#9837]
### What happened?

All of a sudden, I cannot sync. This happened a few days ago...

Debug Log has latest:
 

Few days ago after winning a chia block in a pool, it seemed to have broken syncing

![Capture](https://user-images.githubusercontent.com/69939/149684680-f82edf03-7062-4381-a3a0-9c7d90a349a8.PNG)

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
All of a sudden, I cannot sync. This happened a few days ago...

Debug Log has latest:
 

Few days ago after winning a chia block in a pool, it seemed to have broken syncing


2022-01-15T17:45:53.975 full_node chia.full_node.full_node: INFO     Start syncing from fork point at 0 up to 1427884
2022-01-15T17:45:56.163 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name

or

2022-01-15T19:33:22.077 full_node chia.full_node.full_node: ERROR    Error with syncing: <class 'RuntimeError'>Traceback (most recent call last):
  File "chia\full_node\full_node.py", line 807, in _sync
RuntimeError: Weight proof did not arrive in time from peer: 153.36.77.112

OR


2022-01-15T17:49:46.383 full_node chia.full_node.full_node: INFO     Start syncing from fork point at 0 up to 1427895
2022-01-15T17:49:47.633 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-15T17:49:47.634 full_node chia.full_node.full_node: INFO     long sync done




----


2022-01-16T16:27:59.837 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 1422337-1422369. Traceback (most recent call last):
  File "chia\wallet\wallet_node.py", line 616, in sync_job
  File "chia\wallet\wallet_node.py", line 676, in _sync
  File "chia\wallet\wallet_node.py", line 573, in batch_sync_to_peak
RuntimeError: Was not able to add blocks 1422337-1422369

2022-01-16T16:27:59.837 wallet chia.wallet.wallet_node    : INFO     Loop end in sync job
2022-01-16T16:27:59.837 wallet chia.wallet.wallet_node    : INFO     Loop start in sync job
2022-01-16T16:28:00.098 wallet wallet                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8444}
2022-01-16T16:28:00.098 wallet wallet_server              : INFO     Connected with full_node {'host': '127.0.0.1', 'port': 8444}
2022-01-16T16:28:03.552 full_node full_node_server        : INFO     Connected with full_node {'host': '89.104.10.168', 'port': 8444}
2022-01-16T16:28:04.644 full_node full_node_server        : INFO     Cannot connect to host 101.66.181.70:8444 ssl:<ssl.SSLContext object at 0x000001D2E6217F40> [The semaphore timeout period has expired]
2022-01-16T16:28:04.852 wallet chia.wallet.wallet_node    : INFO     self.sync_event.set()
2022-01-16T16:28:04.853 wallet chia.wallet.wallet_node    : INFO     Have collected 1 potential peaks
2022-01-16T16:28:07.601 full_node full_node_server        : INFO     Connected with full_node {'host': '178.155.240.53', 'port': 8444}
2022-01-16T16:28:07.607 full_node full_node_server        : INFO     Connection closed: 178.155.240.53, node id: d191662d91bf5e6d9495aa7b2c0071d9b2156e73f6dbb434bb47bb56a960962e
2022-01-16T16:28:07.609 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '178.155.240.53', 'port': 8444}
2022-01-16T16:28:07.997 wallet chia.wallet.wallet_node    : INFO     Requesting blocks 1422337-1422369
2022-01-16T16:28:08.001 full_node full_node_server        : INFO     Connection closed: 127.0.0.1, node id: cd8d8625910d6843f0a0a16d51994db89ad812601fbe80b7705cfef613054b4b
2022-01-16T16:28:08.002 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '127.0.0.1', 'port': 8449}
2022-01-16T16:28:08.440 full_node full_node_server        : INFO     Cannot connect to host 116.8.60.193:8444 ssl:<ssl.SSLContext object at 0x000001D2E6EA9040> [The semaphore timeout period has expired]
2022-01-16T16:28:08.001 wallet wallet_server              : INFO     Connection closed: 127.0.0.1, node id: d5045e56a250885fb79abed8d6eecd8bdb97a12237ff593104aa6ebcd36caa94
2022-01-16T16:28:09.008 wallet chia.wallet.wallet_node    : ERROR    Error while trying to fetch from peer:Peer returned no response Traceback (most recent call last):
  File "chia\wallet\wallet_node.py", line 563, in batch_sync_to_peak
  File "chia\wallet\wallet_node.py", line 695, in fetch_blocks_and_validate
ValueError: Peer returned no response

2022-01-16T16:28:09.009 wallet chia.wallet.wallet_node    : ERROR    Loop exception in sync Was not able to add blocks 1422337-1422369. Traceback (most recent call last):
  File "chia\wallet\wallet_node.py", line 616, in sync_job
  File "chia\wallet\wallet_node.py", line 676, in _sync
  File "chia\wallet\wallet_node.py", line 573, in batch_sync_to_peak
RuntimeError: Was not able to add blocks 1422337-1422369

2022-01-16T16:28:09.010 wallet chia.wallet.wallet_node    : INFO     Loop end in sync job
2022-01-16T16:28:09.011 wallet chia.wallet.wallet_node    : INFO     Loop start in sync job
2022-01-16T16:28:09.134 wallet wallet                     : INFO     Reconnecting to peer {'host': '127.0.0.1', 'port': 8444}
2022-01-16T16:28:09.149 wallet wallet_server              : INFO     Connected with full_node {'host': '127.0.0.1', 'port': 8444}


------------------------------------------
```

**Comment by emlowe** [comment]
I initially thought this might have something to do with #9677. But this line is suspect `2022-01-15T17:45:56.163 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name`.

Certainly this implies your DB is in some in-consistent state.

**Comment by deanh2k** [comment]
so how can I fix and correct the issue as this computer has the majority of plots?

**Comment by emlowe** [comment]
What is the value of db_sync in your chia config file `~/.chia/mainnet/config/config.yaml` 

Since it's likely your DB is somehow inconsistent, I recommend resyncing the node db.

stop chia processes
move ~/.chia/mainnet/db/blockchain_v1_mainnet.sqlite out of the way
restart chia

**Comment by deanh2k** [comment]
I don't see db_sync in that chia config file ~/.chia/mainnet/config/config.yaml

in fact, I looked in other computers and don't see that entry.

Could this be due option only coming from 1.2.11 version?

I had other previous versions prior to this version

**Comment by emlowe** [comment]
Not having it is fine, means you get the default setting and didn't change it.

**Comment by lcgogo** [comment]
looks like the same issue

```shell
2022-01-19T05:18:07.557 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:19:55.615 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:22:10.367 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:24:02.383 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:26:05.174 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:28:15.822 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:30:35.237 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:32:36.949 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:34:35.535 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:36:43.611 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:38:44.649 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
2022-01-19T05:40:39.735 full_node chia.full_node.full_node: ERROR    sync from fork point failed err: UNIQUE constraint failed: coin_record.coin_name
```

**Comment by deanh2k** [comment]
okay removed file and its syncing but obviously this is going to take a LONG time..

I was hoping for a way to simply copy the db file from another local machine as a starting point. 
Is that possible?

**Comment by emlowe** [comment]
Yes, you can copy the DB from machine to machine - however, make sure to STOP all chia processes before doing so.

**Comment by deanh2k** [comment]
how do I get the wallet to sync properly?

Copy a wallet db file as well?  IF so what file

**Comment by emlowe** [comment]
Closing issue - this hasn't been reported in some time, so I believe this particular DB corruption has either been fixed in a subsequent release (1.3.1+) or is very rare
